api/notd/token_metadata_processor.py

```diff
@@ -51,7 +51,7 @@ def _resolve_data(dataString: str, registryAddress: str, tokenId: str) -> Dict:
         tokenMetadataJson = None
         if dataString.startswith('data:application/json;base64,'):
             bse64String = dataString.replace('data:application/json;base64,', '', 1)
-            tokenMetadataJson = base64.b64decode(bse64String.encode('utf-8')).decode('utf-8')
+            tokenMetadataJson = base64.b64decode(bse64String.encode('utf-8') + b'==').decode('utf-8')
```

Comment:
this looks dangerous - how sure are you that it doesn't break existing stuff?

Reply:
Well it ignores the excess padding if there is
But i will check it again